tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -298,6 +299,31 @@ export async function resolveOptions(typeSpecDirectory: string): Promise<Exclude
     return options
 }
 
+export function updateApiVersionInTspConfig(typeSpecDirectory: string, apiVersion?: string) {
+    if (!apiVersion || apiVersion === '') {
+        logger.warn(`No apiVersion provided, skipping update to tspconfig.yaml.`);
+        return;
+    }
+    
+    const tspConfigPath = path.join(typeSpecDirectory, 'tspconfig.yaml');
+    if (!fs.existsSync(tspConfigPath)) {
+        throw new Error(`tspconfig.yaml not found at path: ${tspConfigPath}`);
```

Comment:
```suggestion
        throw new Error(`Failed to find tspconfig.yaml at path: ${tspConfigPath}`);
```
Always use `Failed to` to for error or warning log